scripts/deploy.js

```diff
@@ -773,6 +773,7 @@ async function buildUpgradeActionPlan(plan, config) {
     await getRpcUrl(config);
     await getProvider(config);
     await getWallet(config);
+    await getStateBridgeAddress(config);
```

Comment:
Unnecessary in the upgrade. You can remove this call.

Reply:
fixed